pandas/core/frame.py

```diff
@@ -8846,6 +8847,16 @@ def update(
         0  1    4
         1  2  500
         2  3    6
+
+        Updating a dataframe can modify its data types.
```

Comment:
I think there are several issues discussing whether this is the intended behavior, so I would leave this out for now

Reply:
Oh, OK.  Interesting. I'll remove this part from the PR. Thanks!